frame/system/src/mock.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019-2020 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Test utilities
+
+use sp_core::{
```

Comment:
Why can this not be done by `UIntAuthorityId`?

Nevertheless, this file should be moved to somewhere else.

Reply:
This mock is used in `bin/node/executor/tests/submit_transactions.rs` where the runtime used is the actual node runtime where the multisigner is used.

I tried having this code in `bin/node/executor/tests/common` but faced an issue that i'll talk to you offline about.

Reply:
I think it should go into `bin/node/executor/tests ` and it must be a real crypto used by `node-runtime`, not something dummy.

Reply:
Moved